Build failure: libcdio-paranoia

### Nixpkgs version

- Unstable (26.05)

### Steps to reproduce

run 
```bash
 nix-shell -I nixpkgs=channel:nixpkgs-unstable -p cmus
```


### Can Hydra reproduce this build failure?

Hydra doesn’t try to build the package.

### Link to Hydra build job

_No response_

### Relevant log output

```console
username@dust nix-darwin $ nix-shell -I nixpkgs=channel:nixpkgs-unstable -p cmus
these 2 derivations will be built:
  /nix/store/9dq0hrbiwkvwf45kfm7m81k0sp4py2zi-libcdio-paranoia-2.0.2.drv
  /nix/store/jk0fljp0vl2l8qvgny5yib00clszd4hb-cmus-2.12.0.drv
building '/nix/store/9dq0hrbiwkvwf45kfm7m81k0sp4py2zi-libcdio-paranoia-2.0.2.drv'...
Running phase: unpackPhase
unpacking source archive /nix/store/c0c9hglrk9r843g9y7837yfaxxc3v2jr-source
source root is source
Running phase: patchPhase
Running phase: autoreconfPhase
autoreconf: export WARNINGS=
autoreconf: Entering directory '.'
autoreconf: configure.ac: no obvious need to run autopoint
autoreconf: running: aclocal --force -I m4
autoreconf: configure.ac: tracing
autoreconf: running: libtoolize --copy --force
libtoolize: putting auxiliary files in '.'.
libtoolize: copying file './ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4'.
libtoolize: copying file 'm4/libtool.m4'
libtoolize: copying file 'm4/ltoptions.m4'
libtoolize: copying file 'm4/ltsugar.m4'
libtoolize: copying file 'm4/ltversion.m4'
libtoolize: copying file 'm4/lt~obsolete.m4'
autoreconf: configure.ac: not using Intltool
autoreconf: configure.ac: not using Gtkdoc
autoreconf: configure.ac: no need to run autopoint (confirmed)
autoreconf: running: aclocal --force -I m4
autoreconf: running: /nix/store/vhm22v5pqh355py36si9nfn9rifbj8is-autoconf-2.73/bin/autoconf --force
autoreconf: running: /nix/store/vhm22v5pqh355py36si9nfn9rifbj8is-autoconf-2.73/bin/autoheader --force
autoreconf: running: automake --add-missing --copy --force-missing
configure.ac:79: installing './compile'
configure.ac:29: installing './config.guess'
configure.ac:29: installing './config.sub'
configure.ac:27: installing './install-sh'
configure.ac:27: installing './missing'
example/C++/Makefile.am: installing './depcomp'
parallel-tests: installing './test-driver'
autoreconf: configure.ac: not running make: --make not given
autoreconf: Leaving directory '.'
Running phase: updateAutotoolsGnuConfigScriptsPhase
Updating Autotools / GNU config script to a newer upstream version: ./config.sub
Updating Autotools / GNU config script to a newer upstream version: ./config.guess
Running phase: configurePhase
fixing libtool script ./ltmain.sh
./configure
patching script interpreter paths in ./configure
./configure: interpreter directive changed from "#! /bin/sh" to "/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/sh"
configure flags: --disable-static --disable-dependency-tracking --prefix=/nix/store/pk1y2nhhyvcycd5xbvb34x5v7s5nlvgw-libcdio-paranoia-2.0.2 --disable-ld-version-script
checking for a BSD-compatible install... /nix/store/nixxlz2dfdwmy6r8da5sas4nrnj7sq3z-coreutils-9.10/bin/install -c
checking whether sleep supports fractional seconds... yes
checking filesystem timestamp resolution... 0.01
checking whether build environment is sane... yes
checking for a race-free mkdir -p... /nix/store/nixxlz2dfdwmy6r8da5sas4nrnj7sq3z-coreutils-9.10/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking xargs -n works... yes
checking whether UID '351' is supported by ustar format... yes
checking whether GID '350' is supported by ustar format... yes
checking how to create a ustar tar archive... gnutar
checking build system type... aarch64-apple-darwin24.6.0
checking host system type... aarch64-apple-darwin24.6.0
checking whether to enable maintainer-specific portions of Makefiles... no                                                                                                                 checking whether build environment is sane... yes                                                                                                                                          checking for gcc... clang                                                                                                                                                                  checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether the compiler supports GNU C... yes
checking whether clang accepts -g... yes
checking for clang option to enable C23 features... -std=gnu23
checking whether clang -std=gnu23 understands -c and -o together... yes
checking whether make supports the include directive... yes (GNU style)
checking dependency style of clang -std=gnu23... none
checking whether the compiler supports GNU C++... yes
checking whether clang++ accepts -g... yes
checking dependency style of clang++... none
checking for stdio.h... yes                                                                                                                                                                checking for stdlib.h... yes
checking for string.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for strings.h... yes
checking for sys/stat.h... yes
checking for sys/types.h... yes
checking for unistd.h... yes
checking for wchar.h... yes
checking for minix/config.h... no
checking whether it is safe to define __EXTENSIONS__... yes
checking whether _XOPEN_SOURCE should be defined... no
checking for gawk... (cached) gawk
checking for nm... nm
checking whether clang -std=gnu23 understands -Wall... yes
checking whether clang -std=gnu23 understands -Wchar-subscripts... yes
checking whether clang -std=gnu23 understands -Wmissing-prototypes... yes
checking whether clang -std=gnu23 understands -Wmissing-declarations... yes
checking whether clang -std=gnu23 understands -Wunused... yes
checking whether clang -std=gnu23 understands -Wpointer-arith... yes
checking whether clang -std=gnu23 understands -Wwrite-strings... yes
checking whether clang -std=gnu23 understands -Wnested-externs... yes
checking whether clang -std=gnu23 understands -Wno-sign-compare... yes
checking pkg-config is at least version 0.9.0... yes
checking for libcdio >= 2.0.0... yes
checking for perl... false
checking for diff... /nix/store/ckx0pqn935algz20fpw2h588xr012mnh-diffutils-3.12/bin/diff
adding -w to diff in regression tests
adding --unified to diff in regression tests
checking for cmp... /nix/store/ckx0pqn935algz20fpw2h588xr012mnh-diffutils-3.12/bin/cmp
checking for dd... /nix/store/nixxlz2dfdwmy6r8da5sas4nrnj7sq3z-coreutils-9.10/bin/dd
checking for wc... /nix/store/nixxlz2dfdwmy6r8da5sas4nrnj7sq3z-coreutils-9.10/bin/wc
checking for assert.h... yes
checking for errno.h... yes
checking for fcntl.h... yes
checking for glob.h... yes
checking for limits.h... yes
checking for pwd.h... yes
checking for stdarg.h... yes
checking for stdbool.h... yes
checking for stdio.h... (cached) yes
checking for sys/cdio.h... no
checking for sys/param.h... yes
checking for sys/time.h... yes
checking for sys/timeb.h... yes
checking for sys/utsname.h... yes
checking for unistd.h... (cached) yes
checking for getopt.h... yes
checking whether byte ordering is bigendian... no
checking for an ANSI C-conforming const... yes
checking for inline... inline
checking whether clang -std=gnu23 supports ISOC99 _Pragma()... yes
checking for S_ISLNK() macro... yes
checking for S_ISSOCK() macro... yes
checking for struct timespec... yes
checking how to create empty arrays... []
checking how to print strings... printf
checking for a sed that does not truncate output... /nix/store/k6894r7p4545j4sbsyq698pqjwa8vfkh-gnused-4.9/bin/sed
checking for grep that handles long lines and -e... /nix/store/sk1la8r8sa1rlb8m9kwy5hi29f58sjd9-gnugrep-3.12/bin/grep
checking for egrep... /nix/store/sk1la8r8sa1rlb8m9kwy5hi29f58sjd9-gnugrep-3.12/bin/grep -E
checking for fgrep... /nix/store/sk1la8r8sa1rlb8m9kwy5hi29f58sjd9-gnugrep-3.12/bin/grep -F
checking for ld used by clang -std=gnu23... ld
checking if the linker (ld) is GNU ld... no
checking for BSD- or MS-compatible name lister (nm)... nm
checking the name lister (nm) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 786432
checking how to convert aarch64-apple-darwin24.6.0 file names to aarch64-apple-darwin24.6.0 format... func_convert_file_noop
checking how to convert aarch64-apple-darwin24.6.0 file names to toolchain format... func_convert_file_noop
checking for ld option to reload object files... -r
checking for file... file
checking for objdump... objdump
checking how to recognize dependent libraries... (cached) pass_all
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for ranlib... ranlib
checking for archiver @FILE support... @
checking for strip... strip
checking command to parse nm output from clang -std=gnu23 object... ok
checking for sysroot... no
checking for a working dd... /nix/store/nixxlz2dfdwmy6r8da5sas4nrnj7sq3z-coreutils-9.10/bin/dd
checking how to truncate binary pipes... /nix/store/nixxlz2dfdwmy6r8da5sas4nrnj7sq3z-coreutils-9.10/bin/dd bs=4096 count=1
checking for mt... no
checking if : is a manifest tool... no
checking for dsymutil... dsymutil
checking for nmedit... no
checking for lipo... lipo
checking for otool... otool
checking for otool64... no
checking for -single_module linker flag... yes
checking for -no_fixup_chains linker flag... yes
checking for -exported_symbols_list linker flag... yes
checking for -force_load linker flag... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if clang -std=gnu23 supports -fno-rtti -fno-exceptions... yes
checking for clang -std=gnu23 option to produce PIC... -fno-common -DPIC
checking if clang -std=gnu23 PIC flag -fno-common -DPIC works... yes
checking if clang -std=gnu23 static flag -static works... no
checking if clang -std=gnu23 supports -c -o file.o... yes
checking if clang -std=gnu23 supports -c -o file.o... (cached) yes
checking whether the clang -std=gnu23 linker (ld) supports shared libraries... yes
checking dynamic linker characteristics... darwin24.6.0 dyld
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... no
checking how to run the C++ preprocessor... clang++ -E
checking for ld used by clang++... ld
checking if the linker (ld) is GNU ld... no
checking whether the clang++ linker (ld) supports shared libraries... yes
checking for clang++ option to produce PIC... -fno-common -DPIC
checking if clang++ PIC flag -fno-common -DPIC works... yes
checking if clang++ static flag -static works... no
checking if clang++ supports -c -o file.o... yes
checking if clang++ supports -c -o file.o... (cached) yes
checking whether the clang++ linker (ld) supports shared libraries... yes
checking dynamic linker characteristics... darwin24.6.0 dyld
checking how to hardcode library paths into programs... immediate
checking for cos in -lm... yes
checking for clock_gettime in -lrt... no
configure: WARNING: I don't see GNU ld. I'm going to assume --without-versioned-libs
configure: WARNING: Don't have OS CD-reading support for darwin24.6.0...
configure: WARNING: Will use generic support.
checking for bzero... yes
checking for chdir... yes
checking for drand48... yes
checking for ftruncate... yes
checking for geteuid... yes
checking for getgid... yes
checking for getuid... yes
checking for getpwuid... yes
checking for gettimeofday... yes
checking for lstat... yes
checking for memcpy... yes
checking for memset... yes
checking for rand... yes
checking for seteuid... yes
checking for setegid... yes
checking for snprintf... yes
checking for setenv... yes
checking for unsetenv... yes
checking for tzset... yes
checking for sleep... yes
checking for usleep... yes
checking for vsnprintf... yes
checking for readlink... yes
checking for realpath... yes
checking for gmtime_r... yes
checking for localtime_r... yes
checking for clock_gettime... yes
checking that generated files are newer than configure... done
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating doc/Makefile
config.status: creating doc/doxygen/Doxyfile
config.status: creating doc/en/Makefile
config.status: creating doc/en/cd-paranoia.1
config.status: creating doc/ja/Makefile
config.status: creating doc/ja/cd-paranoia.1
config.status: creating example/Makefile
config.status: creating example/C++/Makefile
config.status: creating include/Makefile
config.status: creating include/cdio/Makefile
config.status: creating include/cdio/paranoia/Makefile
config.status: creating include/cdio/paranoia/version.h
config.status: creating lib/Makefile
config.status: creating lib/cdda_interface/Makefile
config.status: creating lib/paranoia/Makefile
config.status: creating libcdio_cdda.pc
config.status: creating libcdio_paranoia.pc
config.status: creating src/Makefile
config.status: creating src/usage.txt
config.status: creating test/data/Makefile
config.status: creating test/cdda_interface/Makefile
config.status: creating test/cdda_interface/toc.c
config.status: creating test/Makefile
config.status: creating test/check_paranoia.sh
config.status: creating test/endian.sh
config.status: creating test/check_start_track_not_one.sh
config.status: creating config.h
config.status: executing depfiles commands
config.status: executing libtool commands
configure:
Building C++ programs: yes
Running phase: buildPhase
build flags: SHELL=/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash
make  all-recursive
make[1]: Entering directory '/nix/var/nix/builds/nix-24144-2366490831/source'
Making all in doc
make[2]: Entering directory '/nix/var/nix/builds/nix-24144-2366490831/source/doc'
Making all in en
make[3]: Entering directory '/nix/var/nix/builds/nix-24144-2366490831/source/doc/en'
make[3]: Nothing to be done for 'all'.
make[3]: Leaving directory '/nix/var/nix/builds/nix-24144-2366490831/source/doc/en'
Making all in ja
make[3]: Entering directory '/nix/var/nix/builds/nix-24144-2366490831/source/doc/ja'
make[3]: Nothing to be done for 'all'.
make[3]: Leaving directory '/nix/var/nix/builds/nix-24144-2366490831/source/doc/ja'
make[3]: Entering directory '/nix/var/nix/builds/nix-24144-2366490831/source/doc'
make[3]: Nothing to be done for 'all-am'.
make[3]: Leaving directory '/nix/var/nix/builds/nix-24144-2366490831/source/doc'
make[2]: Leaving directory '/nix/var/nix/builds/nix-24144-2366490831/source/doc'
Making all in include
make[2]: Entering directory '/nix/var/nix/builds/nix-24144-2366490831/source/include'
Making all in cdio
make[3]: Entering directory '/nix/var/nix/builds/nix-24144-2366490831/source/include/cdio'
Making all in paranoia
make[4]: Entering directory '/nix/var/nix/builds/nix-24144-2366490831/source/include/cdio/paranoia'
make  all-am
make[5]: Entering directory '/nix/var/nix/builds/nix-24144-2366490831/source/include/cdio/paranoia'
make[5]: Nothing to be done for 'all-am'.
make[5]: Leaving directory '/nix/var/nix/builds/nix-24144-2366490831/source/include/cdio/paranoia'
make[4]: Leaving directory '/nix/var/nix/builds/nix-24144-2366490831/source/include/cdio/paranoia'
make[4]: Entering directory '/nix/var/nix/builds/nix-24144-2366490831/source/include/cdio'
make[4]: Nothing to be done for 'all-am'.
make[4]: Leaving directory '/nix/var/nix/builds/nix-24144-2366490831/source/include/cdio'
make[3]: Leaving directory '/nix/var/nix/builds/nix-24144-2366490831/source/include/cdio'
make[3]: Entering directory '/nix/var/nix/builds/nix-24144-2366490831/source/include'
make[3]: Nothing to be done for 'all-am'.
make[3]: Leaving directory '/nix/var/nix/builds/nix-24144-2366490831/source/include'
make[2]: Leaving directory '/nix/var/nix/builds/nix-24144-2366490831/source/include'
Making all in lib
make[2]: Entering directory '/nix/var/nix/builds/nix-24144-2366490831/source/lib'
Making all in cdda_interface
make[3]: Entering directory '/nix/var/nix/builds/nix-24144-2366490831/source/lib/cdda_interface'
/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash ../../libtool  --tag=CC   --mode=compile clang -std=gnu23 -DHAVE_CONFIG_H -I. -I../..  -I../../include -I../../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include   -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c -o common_interface.lo common_interface.c
libtool: compile:  clang -std=gnu23 -DHAVE_CONFIG_H -I. -I../.. -I../../include -I../../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c common_interface.c  -fno-common -DPIC -o .libs/common_interface.o
/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash ../../libtool  --tag=CC   --mode=compile clang -std=gnu23 -DHAVE_CONFIG_H -I. -I../..  -I../../include -I../../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include   -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c -o cddap_interface.lo cddap_interface.c
libtool: compile:  clang -std=gnu23 -DHAVE_CONFIG_H -I. -I../.. -I../../include -I../../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c cddap_interface.c  -fno-common -DPIC -o .libs/cddap_interface.o
/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash ../../libtool  --tag=CC   --mode=compile clang -std=gnu23 -DHAVE_CONFIG_H -I. -I../..  -I../../include -I../../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include   -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c -o interface.lo interface.c
libtool: compile:  clang -std=gnu23 -DHAVE_CONFIG_H -I. -I../.. -I../../include -I../../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c interface.c  -fno-common -DPIC -o .libs/interface.o
/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash ../../libtool  --tag=CC   --mode=compile clang -std=gnu23 -DHAVE_CONFIG_H -I. -I../..  -I../../include -I../../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include   -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c -o scan_devices.lo scan_devices.c
libtool: compile:  clang -std=gnu23 -DHAVE_CONFIG_H -I. -I../.. -I../../include -I../../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c scan_devices.c  -fno-common -DPIC -o .libs/scan_devices.o
/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash ../../libtool  --tag=CC   --mode=compile clang -std=gnu23 -DHAVE_CONFIG_H -I. -I../..  -I../../include -I../../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include   -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c -o smallft.lo smallft.c
libtool: compile:  clang -std=gnu23 -DHAVE_CONFIG_H -I. -I../.. -I../../include -I../../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c smallft.c  -fno-common -DPIC -o .libs/smallft.o
/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash ../../libtool  --tag=CC   --mode=compile clang -std=gnu23 -DHAVE_CONFIG_H -I. -I../..  -I../../include -I../../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include   -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c -o toc.lo toc.c
libtool: compile:  clang -std=gnu23 -DHAVE_CONFIG_H -I. -I../.. -I../../include -I../../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c toc.c  -fno-common -DPIC -o .libs/toc.o
/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash ../../libtool  --tag=CC   --mode=compile clang -std=gnu23 -DHAVE_CONFIG_H -I. -I../..  -I../../include -I../../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include   -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c -o utils.lo utils.c
libtool: compile:  clang -std=gnu23 -DHAVE_CONFIG_H -I. -I../.. -I../../include -I../../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c utils.c  -fno-common -DPIC -o .libs/utils.o
/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash ../../libtool  --tag=CC   --mode=compile clang -std=gnu23 -DHAVE_CONFIG_H -I. -I../..  -I../../include -I../../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include   -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c -o drive_exceptions.lo drive_exceptions.c
libtool: compile:  clang -std=gnu23 -DHAVE_CONFIG_H -I. -I../.. -I../../include -I../../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c drive_exceptions.c  -fno-common -DPIC -o .libs/drive_exceptions.o
/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash ../../libtool  --tag=CC   --mode=link clang -std=gnu23  -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -version-info 2:0:0   -o libcdio_cdda.la -rpath /nix/store/pk1y2nhhyvcycd5xbvb34x5v7s5nlvgw-libcdio-paranoia-2.0.2/lib common_interface.lo cddap_interface.lo interface.lo scan_devices.lo smallft.lo toc.lo utils.lo drive_exceptions.lo  -L/nix/store/21idfh9qzj2lb0kiim3a9y35hhs45mb7-libcdio-2.3.0-lib/lib -lcdio -lm -liconv -Wl,-framework,CoreFoundation -Wl,-framework,IOKit -Wl,-framework,DiskArbitration  -lm -lm
libtool: link: clang -std=gnu23 -dynamiclib -Wl,-undefined -Wl,dynamic_lookup -Wl,-no_fixup_chains -o .libs/libcdio_cdda.2.dylib  .libs/common_interface.o .libs/cddap_interface.o .libs/interface.o .libs/scan_devices.o .libs/smallft.o .libs/toc.o .libs/utils.o .libs/drive_exceptions.o   -L/nix/store/21idfh9qzj2lb0kiim3a9y35hhs45mb7-libcdio-2.3.0-lib/lib /nix/store/21idfh9qzj2lb0kiim3a9y35hhs45mb7-libcdio-2.3.0-lib/lib/libcdio.dylib -liconv -lm  -g -O2 -Wl,-framework -Wl,CoreFoundation -Wl,-framework -Wl,IOKit -Wl,-framework -Wl,DiskArbitration   -install_name  /nix/store/pk1y2nhhyvcycd5xbvb34x5v7s5nlvgw-libcdio-paranoia-2.0.2/lib/libcdio_cdda.2.dylib -compatibility_version 3 -current_version 3.0 -Wl,-single_module
libtool: link: (cd ".libs" && rm -f "libcdio_cdda.dylib" && ln -s "libcdio_cdda.2.dylib" "libcdio_cdda.dylib")
libtool: link: ( cd ".libs" && rm -f "libcdio_cdda.la" && ln -s "../libcdio_cdda.la" "libcdio_cdda.la" )
make[3]: Leaving directory '/nix/var/nix/builds/nix-24144-2366490831/source/lib/cdda_interface'
Making all in paranoia
make[3]: Entering directory '/nix/var/nix/builds/nix-24144-2366490831/source/lib/paranoia'
/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash ../../libtool  --tag=CC   --mode=compile clang -std=gnu23 -DHAVE_CONFIG_H -I. -I../..  -I../../include -I../../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include   -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c -o gap.lo gap.c
libtool: compile:  clang -std=gnu23 -DHAVE_CONFIG_H -I. -I../.. -I../../include -I../../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c gap.c  -fno-common -DPIC -o .libs/gap.o
/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash ../../libtool  --tag=CC   --mode=compile clang -std=gnu23 -DHAVE_CONFIG_H -I. -I../..  -I../../include -I../../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include   -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c -o isort.lo isort.c
libtool: compile:  clang -std=gnu23 -DHAVE_CONFIG_H -I. -I../.. -I../../include -I../../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c isort.c  -fno-common -DPIC -o .libs/isort.o
/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash ../../libtool  --tag=CC   --mode=compile clang -std=gnu23 -DHAVE_CONFIG_H -I. -I../..  -I../../include -I../../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include   -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c -o overlap.lo overlap.c
libtool: compile:  clang -std=gnu23 -DHAVE_CONFIG_H -I. -I../.. -I../../include -I../../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c overlap.c  -fno-common -DPIC -o .libs/overlap.o
/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash ../../libtool  --tag=CC   --mode=compile clang -std=gnu23 -DHAVE_CONFIG_H -I. -I../..  -I../../include -I../../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include   -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c -o p_block.lo p_block.c
libtool: compile:  clang -std=gnu23 -DHAVE_CONFIG_H -I. -I../.. -I../../include -I../../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c p_block.c  -fno-common -DPIC -o .libs/p_block.o
/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash ../../libtool  --tag=CC   --mode=compile clang -std=gnu23 -DHAVE_CONFIG_H -I. -I../..  -I../../include -I../../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include   -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c -o paranoia.lo paranoia.c
libtool: compile:  clang -std=gnu23 -DHAVE_CONFIG_H -I. -I../.. -I../../include -I../../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c paranoia.c  -fno-common -DPIC -o .libs/paranoia.o
paranoia.c:717:8: warning: variable 'tried' set but not used [-Wunused-but-set-variable]
  717 |   long tried = 0;
      |        ^
paranoia.c:718:8: warning: variable 'matched' set but not used [-Wunused-but-set-variable]
  718 |   long matched = 0;
      |        ^
2 warnings generated.
/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash ../../libtool  --tag=CC   --mode=link clang -std=gnu23  -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -version-info 2:0:0   -o libcdio_paranoia.la -rpath /nix/store/pk1y2nhhyvcycd5xbvb34x5v7s5nlvgw-libcdio-paranoia-2.0.2/lib gap.lo isort.lo overlap.lo p_block.lo paranoia.lo  -L/nix/store/21idfh9qzj2lb0kiim3a9y35hhs45mb7-libcdio-2.3.0-lib/lib -lcdio -lm -liconv -Wl,-framework,CoreFoundation -Wl,-framework,IOKit -Wl,-framework,DiskArbitration ../../lib/cdda_interface/libcdio_cdda.la
libtool: link: clang -std=gnu23 -dynamiclib -Wl,-undefined -Wl,dynamic_lookup -Wl,-no_fixup_chains -o .libs/libcdio_paranoia.2.dylib  .libs/gap.o .libs/isort.o .libs/overlap.o .libs/p_block.o .libs/paranoia.o   -L/nix/store/21idfh9qzj2lb0kiim3a9y35hhs45mb7-libcdio-2.3.0-lib/lib ../../lib/cdda_interface/.libs/libcdio_cdda.dylib /nix/store/21idfh9qzj2lb0kiim3a9y35hhs45mb7-libcdio-2.3.0-lib/lib/libcdio.dylib -liconv -lm  -g -O2 -Wl,-framework -Wl,CoreFoundation -Wl,-framework -Wl,IOKit -Wl,-framework -Wl,DiskArbitration   -install_name  /nix/store/pk1y2nhhyvcycd5xbvb34x5v7s5nlvgw-libcdio-paranoia-2.0.2/lib/libcdio_paranoia.2.dylib -compatibility_version 3 -current_version 3.0 -Wl,-single_module
libtool: link: (cd ".libs" && rm -f "libcdio_paranoia.dylib" && ln -s "libcdio_paranoia.2.dylib" "libcdio_paranoia.dylib")
libtool: link: ( cd ".libs" && rm -f "libcdio_paranoia.la" && ln -s "../libcdio_paranoia.la" "libcdio_paranoia.la" )
make[3]: Leaving directory '/nix/var/nix/builds/nix-24144-2366490831/source/lib/paranoia'
make[3]: Entering directory '/nix/var/nix/builds/nix-24144-2366490831/source/lib'
make[3]: Nothing to be done for 'all-am'.
make[3]: Leaving directory '/nix/var/nix/builds/nix-24144-2366490831/source/lib'
make[2]: Leaving directory '/nix/var/nix/builds/nix-24144-2366490831/source/lib'
Making all in src
make[2]: Entering directory '/nix/var/nix/builds/nix-24144-2366490831/source/src'
cp usage-copy.h usage.h
clang -std=gnu23 -DHAVE_CONFIG_H -I. -I..  -I.. -I../include -I../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include   -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c -o cd-paranoia.o cd-paranoia.c
In file included from cd-paranoia.c:63:
./getopt.h:169:12: error: conflicting types for 'getopt'
  169 | extern int getopt ();
      |            ^
/nix/store/rcqgjj8hphkhqark1ibiwfaa7yrzniz3-apple-sdk-14.4/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/unistd.h:509:6: note: previous declaration is here
  509 | int      getopt(int, char * const [], const char *) __DARWIN_ALIAS(getopt);
      |          ^
1 error generated.
make[2]: *** [Makefile:466: cd-paranoia.o] Error 1
make[2]: Leaving directory '/nix/var/nix/builds/nix-24144-2366490831/source/src'
make[1]: *** [Makefile:516: all-recursive] Error 1
make[1]: Leaving directory '/nix/var/nix/builds/nix-24144-2366490831/source'
make: *** [Makefile:421: all] Error 2
error: Cannot build '/nix/store/9dq0hrbiwkvwf45kfm7m81k0sp4py2zi-libcdio-paranoia-2.0.2.drv'.
       Reason: builder failed with exit code 2.
       Output paths:
         /nix/store/pk1y2nhhyvcycd5xbvb34x5v7s5nlvgw-libcdio-paranoia-2.0.2
       Last 25 log lines:
       > libtool: link: clang -std=gnu23 -dynamiclib -Wl,-undefined -Wl,dynamic_lookup -Wl,-no_fixup_chains -o .libs/libcdio_paranoia.2.dylib  .libs/gap.o .libs/isort.o .libs/overlap.o .libs/p_block.o .libs/paranoia.o   -L/nix/store/21idfh9qzj2lb0kiim3a9y35hhs45mb7-libcdio-2.3.0-lib/lib ../../lib/cdda_interface/.libs/libcdio_cdda.dylib /nix/store/21idfh9qzj2lb0kiim3a9y35hhs45mb7-libcdio-2.3.0-lib/lib/libcdio.dylib -liconv -lm  -g -O2 -Wl,-framework -Wl,CoreFoundation -Wl,-framework -Wl,IOKit -Wl,-framework -Wl,DiskArbitration   -install_name  /nix/store/pk1y2nhhyvcycd5xbvb34x5v7s5nlvgw-libcdio-paranoia-2.0.2/lib/libcdio_paranoia.2.dylib -compatibility_version 3 -current_version 3.0 -Wl,-single_module
       > libtool: link: (cd ".libs" && rm -f "libcdio_paranoia.dylib" && ln -s "libcdio_paranoia.2.dylib" "libcdio_paranoia.dylib")
       > libtool: link: ( cd ".libs" && rm -f "libcdio_paranoia.la" && ln -s "../libcdio_paranoia.la" "libcdio_paranoia.la" )
       > make[3]: Leaving directory '/nix/var/nix/builds/nix-24144-2366490831/source/lib/paranoia'
       > make[3]: Entering directory '/nix/var/nix/builds/nix-24144-2366490831/source/lib'
       > make[3]: Nothing to be done for 'all-am'.
       > make[3]: Leaving directory '/nix/var/nix/builds/nix-24144-2366490831/source/lib'
       > make[2]: Leaving directory '/nix/var/nix/builds/nix-24144-2366490831/source/lib'
       > Making all in src
       > make[2]: Entering directory '/nix/var/nix/builds/nix-24144-2366490831/source/src'
       > cp usage-copy.h usage.h
       > clang -std=gnu23 -DHAVE_CONFIG_H -I. -I..  -I.. -I../include -I../include/ -I/nix/store/9ch59r5mk70bn0jr7sxyagj9knwpsnrf-libcdio-2.3.0-dev/include   -g -O2 -Wall -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wunused -Wpointer-arith -Wwrite-strings -Wnested-externs -Wno-sign-compare -c -o cd-paranoia.o cd-paranoia.c
       > In file included from cd-paranoia.c:63:
       > ./getopt.h:169:12: error: conflicting types for 'getopt'
       >   169 | extern int getopt ();
       >       |            ^
       > /nix/store/rcqgjj8hphkhqark1ibiwfaa7yrzniz3-apple-sdk-14.4/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/unistd.h:509:6: note: previous declaration is here
       >   509 | int      getopt(int, char * const [], const char *) __DARWIN_ALIAS(getopt);
       >       |          ^
       > 1 error generated.
       > make[2]: *** [Makefile:466: cd-paranoia.o] Error 1
       > make[2]: Leaving directory '/nix/var/nix/builds/nix-24144-2366490831/source/src'
       > make[1]: *** [Makefile:516: all-recursive] Error 1
       > make[1]: Leaving directory '/nix/var/nix/builds/nix-24144-2366490831/source'
       > make: *** [Makefile:421: all] Error 2
       For full logs, run:
         nix log /nix/store/9dq0hrbiwkvwf45kfm7m81k0sp4py2zi-libcdio-paranoia-2.0.2.drv
error: Cannot build '/nix/store/jk0fljp0vl2l8qvgny5yib00clszd4hb-cmus-2.12.0.drv'.
       Reason: 1 dependency failed.
       Output paths:
         /nix/store/78f4bfjb48dl326p1ifv1bhrbwa17fsr-cmus-2.12.0
error: Build failed due to failed dependency
username@dust nix-darwin $
```

### Additional context

_No response_

### System metadata

nix-shell -p nix-info --run "nix-info -m"
 - system: `"aarch64-darwin"`
 - host os: `Darwin 24.6.0, macOS 15.7.4`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.34.6+1`
 - channels(root): `"nixpkgs"`
 - nixpkgs: `/nix/store/i2gsp87gqp16whm9mw0ybk9n84zir01x-source`

### Notify maintainers


@oxij 
---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+label%3A%220.kind%3A+build+failure%22).
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/li/libcdio-paranoia/package.nix
+++ b/pkgs/by-name/li/libcdio-paranoia/package.nix
@@ -30,7 +30,10 @@ stdenv.mkDerivation (finalAttrs: {
     libiconv
   ];
 
-  configureFlags = lib.optionals stdenv.hostPlatform.isDarwin [ "--disable-ld-version-script" ];
+  configureFlags = [
+    "CFLAGS=-std=gnu17"
+  ]
+  ++ lib.optionals stdenv.hostPlatform.isDarwin [ "--disable-ld-version-script" ];
 
   meta = {
     description = "CD paranoia on top of libcdio";

--- a/pkgs/by-name/li/libcdio-paranoia/package.nix
+++ b/pkgs/by-name/li/libcdio-paranoia/package.nix
@@ -32,8 +32,7 @@ stdenv.mkDerivation (finalAttrs: {
 
   configureFlags = [
     "CFLAGS=-std=gnu17"
-  ]
-  ++ lib.optionals stdenv.hostPlatform.isDarwin [ "--disable-ld-version-script" ];
+  ];
 
   meta = {
     description = "CD paranoia on top of libcdio";

